packages/docusaurus-theme-classic/src/theme/CodeBlock/Content/styles.module.css

```diff
@@ -84,3 +84,7 @@
 :global(.theme-code-block:hover) .buttonGroup button {
   opacity: 0.4;
 }
+
+.buttonBlockSpace {
+  right: calc(var(--ifm-pre-padding) / 2) !important;
+}
```

Comment:
That's not a good fix because our RTLCSS postCSS plugin will automatically transform `right` to `left`.

If you test your dev locally `yarn start:website --locale fa` you'll notice it doesn't work.

The solution is to add an rtl ignore directive to avoid this auto transform to happen:

```css
.buttonGroup {
  display: flex;
  column-gap: 0.2rem;
  position: absolute;
  /* rtl:ignore */
  right: calc(var(--ifm-pre-padding) / 2);
  top: calc(var(--ifm-pre-padding) / 2);
}
```